fireant/slicer/queries/logger.py

```diff
@@ -1,3 +1,7 @@
 import logging
 
-logger = logging.getLogger('fireant.query_log$')
+SLOW_QUERY_LOG_MIN_DURATION = 15
```

Comment:
this should probably be an attribute in the Database class instead of a constant defined in an obscure place, that way it can be configured easily

Reply:
I set it in the logger module as it associated with the logging.

Nevertheless, I have moved it to the database module now.